core/templates/components/shared-component.module.ts

```diff
@@ -206,8 +214,15 @@ import { HybridRouterModuleProvider } from 'hybrid-router-module-provider';
     SortByPipe,
     LearnerDashboardIconsComponent,
     PreviewThumbnailComponent,
+    AddHintModalComponent,
+    DeleteInteractionModalComponent,
+    DeleteHintModalComponent,
+    DeleteLastHintModalComponent,
+    DeleteSolutionModalComponent,
     SavePendingChangesModalComponent,
-    AddHintModalComponent
+    HintEditorComponent,
+    ResponseHeaderComponent,
+    StateHintsEditorComponent
```

Comment:
Keep these sorted

Reply:
@JeeveshGarg PTAL too.

Reply:
Done in mine

Reply:
@ashutoshc8101 in declarations, entryComponents and exports, files entries are already in unsorted manner, so i add them in bottom, do i need to sort all along with my.

@heyimShivam after above confirmation you can take a pull.